### DISCUSSION TITLE
Schema and ZodSchema undocumented

### DISCUSSION THREAD

**Discussion by Gpx** [colinhacks/zod#1963] [Q&A] [answered]
Looking at the code, I can see that `ZodType` is aliased as `ZodSchema` and `Schema`: https://github.com/colinhacks/zod/blob/master/src/types.ts#L4309

```ts
export { ZodType as Schema, ZodType as ZodSchema };
```

But I see no mention of this in the documentation. Are those aliases meant to be used? If so, should the documentation reflect that?

**Comment by JacobWeisenburger** [comment, marked as answer]
I think they can be used. Where would be a good place to show this in the docs?

**Reply by Gpx** [reply]
Probably [here](https://zod.dev/?id=recursive-types) where `ZodType` is first mentioned. Although I'm not sure it makes sense to keep these aliases, I think they could be cofusing.
